operationalGuide.md

```diff
@@ -0,0 +1,174 @@
+
+* *Id: technical identifier for an instance of a resource, is always in the uuid format; is used for direct access to a resource instance in an API endpoint
+* *Reference: functional identifier for an instance of a resource that can also be known by an end customer, is a generic string without a standardized format 
+* *Code: identifier for an instance of a resource, can be mapped to a name in a finite set of values, is a generic string without a standardized format
+
+
+# Card API Level 1
+
+## Model
+![Card-API_Level1](https://github.com/swissfintechinnovations/ca-card/assets/141315380/7ec14f1a-7fa8-4cdb-bca2-8f806f0e843c)
+
+### “cards” resource
+The term “cards” refers to both physical and virtual credit or debit cards issued by banks or financial institutions. Physical cards are the traditional plastic cards that holders carry with them. Virtual cards, on the other hand, only exist in digital form and are often used for online payments. These virtual cards have a card number, expiration date and security code like physical cards.
+### “card-tokens” resource
+The term “card tokens” refers to the DPAN (Digital Primary Account Number) of a credit or debit card that is stored in digital wallets, wearables or with merchants (e.g. for click-to-pay). The “card token” serves as a replacement for the PAN and is used for transactions instead of the real card number. This reduces the risk of data misuse or fraud, as the sensitive card data is not transmitted directly.
+### “transactions” resource
+The term “transactions” refers to payments made with physical or virtual cards or with card tokens. These transactions are displayed on the cardholder's account and include all authorized and recorded payments entered by the cardholder.
+
+Transactions can include various types of payments, such as retail purchases, online payments, transfers between accounts, etc. Each transaction is authorized and recorded, which means that it has been approved by the cardholder and the payment amount has been debited from the cardholder's account.
+
+## Use Cases
+| Identifier | Description |
+| -------- | ------- |
+| PFM/BFM | Aggregation of transactions for cards from different issuers to make evaluations on spending behavior, budget, ecological footprint or more.<br>Limitation: Account-based transactions are not taken into account.<br>Challenge with these endpoints: incomplete view of card account transactions. |
+| Card overview| Aggregation of cards and associated tokens from different issuers for display.<br>Restriction: Structure of cards in an account or customer relationship is not easily traceable. |
+
+## Remarks
+none so far
+
+
+# Card API Level 2
+
+## Model
+![Card-API_Level12png](https://github.com/swissfintechinnovations/ca-card/assets/141315380/649d1704-f7ae-4821-bb69-abb9450e99a4)
+
+## Use Cases
```

Comment:
- That is added value to level 1?
- In my opinion, more important Use Cases are missing, as e.g.: client administration: client can manage all its cards

Reply:
- Current focus for the Card API are read-only use cases. Mutation cases might be added in the future if there is concrete demand.